Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: test test-verbose test-profile clean docker bootstrap
+
+PACKAGES = `go list ./... | grep -v vendor/`
+
+all: inertia
+
+inertia:
+	go build
+
+test:
+	go test $(PACKAGES) --cover
```

Comment:
Might as well add -race

Reply:
TIL gotest has a race condition detector 😱 

Speaking of which need to make sure daemon doesn't do bad things when a bunch up people ping it to do things at the same time